templates/primarypersonlistaddoreditquestion.html

```diff
@@ -0,0 +1 @@
+{% extends 'question.html' %}
```

Comment:
If not overriding anything, should this be an include too, in common with primarypersonlistcollector.html?